### ISSUE TITLE
--enable-linuxgpio creates wrong define

### ISSUE THREAD

**Issue by thorsteneisbein** [kcuzner/avrdude#14]
If using --enable-linuxgpio the ./configure script does:
$as_echo "#define HAVE_LINUXGPIO 1" >>confdefs.h

In spidev.c you use HAVE_LINUX_GPIO. One of them is wrong.

**Comment by smvoss** [comment]
Pull request created to fix this. See #15

**Comment by kcuzner** [comment]
There is no `spidev.c`. To which file do you refer?

Also, it does not appear that this change is related to the `linuxspi` programmer type. I just checked the log and the only files that have been modified by me as part of this programmer type are:
```avrdude/avrdude.conf.in
avrdude/configure.ac
avrdude/doc/avrdude.texi
avrdude/.gitignore
avrdude/linuxspi.c
avrdude/linuxspi.h
avrdude/Makefile.am
avrdude/pgm_type.c
```

I don't recall any other files being modified as a result of pull requests, but those go back four years or so. In any case, none of these files make reference to the `HAVE_LINUXGPIO` define. I did find a typo in `pindefs.h` referencing `HAVE_LINUX_GPIO` which has not been modified as part of the `linuxspi` changes. This file is part of the [mainline avrdude](http://svn.savannah.gnu.org/viewvc/avrdude/trunk/avrdude/) (managed with SVN on GNU savannah) and any changes to those files should be submitted there in order to keep this repository specific to the `linuxspi` programmer type.

Note that `pindefs.h` no longer exists in the mainline SVN repository, as they seem to have moved on from it. `pindefs.c` still does exist.

**Comment by smvoss** [comment]
Please see my comment on #15 as to why this change is important for your fork, even if it was not modified by you.

**Comment by kcuzner** [comment]
Your fix in #15 should fix the problem and I've merged it, though it would be nice if the OP could comment if it works.
